client/src/app/tabs/form/__tests__/FormEditorSpec.js

```diff
@@ -622,7 +622,14 @@ describe('<FormEditor>', function() {
       // then
       const { form } = instance.getCached();
 
-      expect(onActionSpy).to.have.been.calledOnce;
+      var calls = onActionSpy.getCalls();
+
+      calls = calls.filter(function(call) {
+        return call.args[0] === 'lint-tab';
+      });

```

Comment:
In Camunda Modeler, we use ES6 syntax:
```suggestion
      const calls = onActionSpy.getCalls()
        .filter(call => call.args[0] === 'lint-tab');
```